src/System.Net.Http/tests/FunctionalTests/HttpClientMiniStressTest.cs

```diff
@@ -0,0 +1,128 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Http.Functional.Tests
+{
+    public class HttpClientMiniStress
+    {
+        // [Theory] // Too stressful for CI; uncomment to run locally
```

Comment:
So, these aren't really [OuterLoop]?  It would be nice if there was a way to simply define an environment variable and then the tests could run.  So, perhaps this could be a [ConditionalTheory]?  It's anonying having to change the source code just to run the tests.  Maybe have HTTP_STRESSTEST=1 to have the tests run?